governance/votes/Executive vote - July 11, 2023.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Template - [Executive Vote] Stability Scope Parameter Changes, Smart Burn Engine Deployment, DssCron Housekeeping, Various Budget Streams and Payments - July 11, 2023
+summary: Deploy Smart Burn Engine, Change Stability Scope parameters, set up payments and streams to Aligned Delegates, Chronicle Labs and DECO, DssCron Housekeeping
+date: $2023-07-11T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] Stability Scope Parameter Changes, Smart Burn Engine Deployment, DssCron Housekeeping, Various Budget Streams and Payments - July 11, 2023
+
+The $exec_publishers and $exec_crafters have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur within the Maker Protocol:
+- The Smart Burn Engine will be deployed and the old MCD_Flapper will be disabled
+- The Dai Savings Rate will be reduced from 3.49% to 3.19%
+- Multiple risk parameters will be changed including Stability Fees, Maximum Debt Ceilings and Liquidation Ratios, as detailed below.
+- DssCron Housekeeping Additions
+
+If this executive proposal passes, the following **Actions** will occur within Maker Protocol:
+- Aligned Delegate One-Off Compensation totaling **X MKR** will be distributed
+- 125 MKR will be transfered to DECO-001
+- DAI and MKR streams will be set up for Chronicle Labs, as detailed below.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. 
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### Smart Burn Engine Deployment
+
+As per [this poll](https://vote.makerdao.com/polling/QmQmxEZp), the Smart Burn Engine will be implemented and the old flapper Depreciated. The following actions will occur
+
+- Depreciate MCD_FLAP
+- Deploy [FlapperUniV2](https://github.com/makerdao/dss-flappers/blob/univ2/README.md)
+- Reduce vow.hump by 200 million DAI from 250 million DAI to 50 million DAI
```

Comment:
In general, don't just copy directly from the executive sheet. The requirements and audience of that sheet are not the same as the requirements and audience for this copy.

Reply:
Also its important to note that this parameter change is not happening because of the poll on the new Flapper, it's primarily happening because its in the scope.